error_reporting/google/cloud/error_reporting/client.py

```diff
@@ -12,13 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Client for interacting with the Stackdriver Logging API"""
+"""Client for interacting with the Stackdriver Error Reporting API"""
 
+import os
 import traceback
 
-import google.cloud.logging.client
+try:
```

Comment:
I mostly just copied this structure from `logging/client.py`. I can look into a bit more where exactly the import fails, I assume on one of the grpc imports in `_gax.py`.